content/post/52-go-ipfs-0.4.18.md

```diff
@@ -0,0 +1,283 @@
+but here's a quick example of connecting the WebUI of a remote IPFS node:
+
+```sh
+# On the 'remote' IPFS host
+> ipfs id -f "<id>\n"
+QmSiXjrYwLmDhRvAb3vK2TUP8W2pTDd34MhgCwpanVjdNT
+
+# Configure the p2p listener on the remote IPFS instance:
+> ipfs p2p listen /x/kickass/1.0 /ip4/127.0.0.1/tcp/5001
+
+# On the 'local' IPFS host
+# Configure the p2p forwarder on the local host:
+> ipfs p2p forward /x/kickass/1.0 /ip4/127.0.0.1/tcp/5551 /ipfs/QmSiXjrYwLmDhRvAb3vK2TUP8W2pTDd34MhgCwpanVjdNT
+
+# Voila - point your browser at http://localhost:5551/webui to inspect your remote
+```
+
+#### `ipfs name resolve` streaming response
+
+There is now a new flag for `ipfs name resolve` - `--stream`. When the command
+is invoked with the flag set, it will start returning results as soon as they
+are discovered in the DHT and other routing mechanisms. This enables certain
+applications to start prefetching/displaying data while the discovery is still
+running. Note that this command will likely return many outdated records
+before it finding and returning the latest. However, it will always return
+*valid* records (even if a bit stale).
+
+#### `ipfs add` block inlining
+
+In the previous release, we added support for extracting blocks inlined
+into CIDs. In this release, we've added support for creating these CIDs. You can
+now run `ipfs add` with the `--inline` flag to inline blocks less than or equal
+to 32 bytes in length into a CID, instead of writing an actual block. This
+should significantly reduce the size of filesystem trees with many empty
+directories and tiny files.
+
+## WebUI
+
+This release includes the latest, very shiny [updated webui](https://github.com/ipfs-shipyard/ipfs-webui). You can view it by
+installing go-ipfs and visiting http://localhost:5001/webui. It deserves its own release note - oh look, it [got
+one](./51-js-ipfs-0-33/#web-ui-2-0)! Here's a peek:
+
+![Screenshot of the status page](https://raw.githubusercontent.com/ipfs-shipyard/ipfs-webui/master/docs/screenshots/ipfs-webui-status.png)
+
+| Files | Explore | Peers | Settings |
+|-------|---------|-------|----------|
+| ![Screenshot of the file browser page](https://raw.githubusercontent.com/ipfs-shipyard/ipfs-webui/master/docs/screenshots/ipfs-webui-files.png) | ![Screenshot of the IPLD explorer page](https://raw.githubusercontent.com/ipfs-shipyard/ipfs-webui/master/docs/screenshots/ipfs-webui-explore.png) | ![Screenshot of the swarm peers map](https://raw.githubusercontent.com/ipfs-shipyard/ipfs-webui/master/docs/screenshots/ipfs-webui-peers.png) | ![Screenshot of the settings page](https://raw.githubusercontent.com/ipfs-shipyard/ipfs-webui/master/docs/screenshots/ipfs-webui-settings.png) |
+
+
+Kudos and thanks to the webui team!
+
+## Performance
+
+This release includes some significant performance improvements, both in terms
+of resource utilization and speed. This section will go into some technical
```

Comment:
Let's add a line about the value of the performance improvements for the folks that will skip the "technical details". 
> **Ex:** We're now being more efficient with our CPU and memory usage through a number of optimizations to our data structures, stream management, and bulk message passing - making it faster to transfer small files or manage many connections.